syft/frameworks/crypten/context.py

```diff
@@ -83,15 +85,7 @@ def _send_party_info(worker, rank, msg, return_values):
     return_values[rank] = response.contents
 
 
-def toy_func():
-    alice_tensor = syft_crypt.load("crypten_data", 1, "alice")
-    bob_tensor = syft_crypt.load("crypten_data", 2, "bob")
-
-    crypt = crypten.cat([alice_tensor, bob_tensor], dim=0)
-    return crypt.get_plain_text().tolist()
-
-
-def run_multiworkers(workers: list, master_addr: str, master_port: int = 15987):
+def run_multiworkers(plan, workers: list, master_addr: str, master_port: int = 15995):
```

Comment:
Shouldn't the plan be transparent here? Like the user would only decorate his function and we build a plan out of his function

Reply:
Yep, it should. I added it like this to have a fast testing mechanism if the plans work,